.env.test

```diff
@@ -0,0 +1,16 @@
+DEBUG=true
+SECRET_KEY='tests'
+DATABASE_URL='postgresql://postgres@localhost/opencraft'
```

Comment:
This doesn't work for me:
```
django.db.utils.OperationalError: fe_sendauth: no password supplied
```

I would suggest making this default to `DATABASE_URL='postgresql:///opencraft'`

And add a comment that the developer just has to run the one-time setup command `sudo -u postgres createuser -d opencraft`, where 'opencraft' is replaced with the name of whatever user the app runs under.